deps/v8/src/base/build_config.h

```diff
@@ -203,11 +203,7 @@ constexpr int kReturnAddressStackSlotCount =
     V8_TARGET_ARCH_STORES_RETURN_ADDRESS_ON_STACK ? 1 : 0;
 
 // Number of bits to represent the page size for paged spaces.
-#if defined(V8_TARGET_ARCH_PPC) || defined(V8_TARGET_ARCH_PPC64)
-// PPC has large (64KB) physical pages.
-const int kPageSizeBits = 19;
-#else
+// Use 4KB for all targets. Not all PPC Linux kernels use 64KB pages.
 const int kPageSizeBits = 18;
```

Comment:
On newer versions of P Linux, page size needs to remain as 64KB as it's the default size of the OS.
AIX and older Linux versions use 4KB by default. This is causing some V8 test failures on P Linux.

Reply:
modern Linux allows both 4k and 64k page sizes, but the constant name is misleading - using 64k there will work on both 4k and 64k hosts, while using 4k there will break 64k hosts